File: 2023/Primeira Fase/Contas a Pagar/contas.cpp
#include <bits/stdc++.h>

using namespace std;

int main(){
    int valor, contas[3];

    cin >> valor;

    for(int i=0; i<3; i++){
        cin >> contas[i];
    }

    sort(contas, contas+3);

    int resp=0;

    for(int i=0; i<3; i++){
        if(valor>=contas[i]){
            valor-=contas[i];
            resp++;
        }
    }
    cout << resp << endl;
    return 0;
}

File: 2023/Primeira Fase/Contas a Pagar/contas.py
valor = int(input())

contas = []

for i in range(3):
    conta = int(input())
    contas.append(conta)

contas = sorted(contas)

resp = 0

for conta in contas:
    if valor>=conta:
        valor-=conta
        resp+=1

print(resp)
